apps/sms/views/inbox/js/startup.js

```diff
@@ -47,6 +47,10 @@
         initLazyDependencies();
       });
 
+      InboxView.once('fully-loaded', () => {
+        App.setReady();
```

Comment:
Good, we have tests for the split view mode startup.js  - so I'll add one for the case as well, thanks!